cmd/zpool/zpool_main.c

```diff
@@ -2836,12 +3393,263 @@ get_timestamp_arg(char c)
 }
 
 /*
- * zpool iostat [-gLPv] [-T d|u] [pool] ... [interval [count]]
+ * Return stat flags that are supported by all pools.  "data" should be
+ * initialized to all 0xFFs before running.  It will get ANDed down until
+ * only the flags that are supported on all pools remain.
+ */
+static int
+__get_stat_flags(zpool_handle_t *zhp, void *data)
```

Comment:
Often in the existing code functions which are used as callback use a `_cb` suffix.  So in this case I'd suggest calling this `get_stat_flags_cb()`.